surveys/sphinx-survey-2025.yml

```diff
@@ -53,25 +53,13 @@ questions:
   - "Academic researcher"
   - "Data scientist"
   - "Manager"
-  - "ML engineer / MLOps"
   - "Product manager"
-  - "QA engineer"
   - "Software Developer / Programmer / Engineer"
   - "Student (in full-time education)"
   - "Technical support"
   - "Technical writer"
   - "Prefer not to answer"
 
-# Motivation: Understand the responder's self-reported background and skills.
-- title: "Do you most often work in a team or independently?"
-  other: true
-  choices:
-  - "I most often work on projects independently"
-  - "I most often work on projects as part of a team"
-  - "I spend about equal time working on projects independently as I do on a
-    team"
-  - "Prefer not to answer"
-
```

Comment:
I do not think there were intentions to remove this. Can we restore the question?

Reply:
I can restore it. No one specifically asked for this removed, I removed it on general feedback on the length of the survey.